UI/src/view/point_asset/massTransitStopForm.js

```diff
@@ -67,14 +75,31 @@
           }
         });
       } else {
-          if(selectedMassTransitStopModel.validateDirectionsForSave()){
-              selectedMassTransitStopModel.save();
-          }else{
-              new GenericConfirmPopup('Pysäkin vaikutussuunta on yksisuuntaisen tielinkin ajosuunnan vastainen. Pysäkkiä ei tallennettu.',
-                  {type: 'alert'});
-          }
+        if(optionalSave(selectedMassTransitStopModel.getProperties())){
+          new GenericConfirmPopup('Oletko varma, ettet halua lähettää pysäkin tietoja Tierekisteriin? Jos vastaat kyllä, tiedot tallentuvat ainoastaan OTH-sovellukseen', {
+            successCallback: function () {
+              selectedMassTransitStopModel.setAdditionalProperty('trSave', [{ propertyValue: 'false' }]);
+              saveStop();
+            },
+            closeCallback: function () {
+              saveStop();
+            }
+          });
+        } else {
+          saveStop();
```

Comment:
The `saveStop()` call can not be done on the end of everything instead of being inside of `successCallback`, `closeCallback` and the `else ` ?

Reply:
Maybe in some way, but now if its outside of all brackets it calls the savestop() before answering.